ShipperOptions.json

```diff
@@ -1796,6 +1796,10 @@
       {
         "value": "RBB",
         "display": "Regional Rate Box B"
+      },
+      {
+        "value": "Flat",
```

Comment:
in skipper it says `flat = 'FLAT'`. is it Flat, or FLAT?

Reply:
should be FLAT